### ISSUE TITLE
When hosting CrimeSpree, crew can not join.

### ISSUE THREAD

**Issue by katabame** [SydneyMOD/SydneyHUD#14]
from Steam [Bug Report Thread](http://steamcommunity.com/groups/SydneyMOD/discussions/4/133262738419706698/)

**Comment by katabame** [comment]
Crashlog:
```
 [string "lib/managers/missionassetsmanager.lua"]:551: attempt to index field '?' (a nil value)
```

**Comment by katabame** [comment]
[Kamikaze94/WolfHUD/#228](https://github.com/Kamikaze94/WolfHUD/issues/228) is maybe similar issue.
Import some codes from [Kamikaze94/WolfHUD/d4b0d75](https://github.com/Kamikaze94/WolfHUD/commit/d4b0d7522fe3600d6739e94c7bb8b635ae8bb53d#diff-5079b39528e730f86271efb7a5317108) will fix this.

**Comment by katabame** [comment]
Fixed in 331e731fe6bef301af2dcf3ccb90b2c70598a9aa
